### ISSUE TITLE
creating transcript isoform-specific annotations

### ISSUE THREAD

**Issue by mah11** [pombase/canto#2351]
Now that we have a few real live "alternative" transcripts annotated, it's time to turn to what Canto does with them. We already have cases where different transcript isoforms or their protein products should get different annotations (examples below), some of which could be done in Canto if we had a way to distinguish them. I guess the thing to do would be to enter transcript systematic IDs in the gene list so we can use them in annotations and extensions.

- examples that could be done in Canto once this ticket is done:
  - GO CCs for wtf4 (I just stuck in comments for now, but will go back and assign CCs to the correct isoforms; session )
  - tco1 transcripts in extensions on sre1 phenotypes

- another example that would be Canto-able soon: tco1 RNA levels (gene expression)

At this point I'm mainly opening this for discussion and so it won't fall off the radar. ~I'm about to open~ I opened a [website ticket](https://github.com/pombase/website/issues/1613) about displays,  ~Will we also need~ and [a Chado ticket](https://github.com/pombase/pombase-chado/issues/795) to replace the [old one](https://github.com/pombase/pombase-chado/issues/236) that we closed because there was no need to do anything at the time.

**Comment by ValWood** [comment]
Or, if a gene product  which has isoforms annotated is entered into Canto , when you make a GO annotation there is automagically an extension option  with the transcript IDs available (more similar to the protein modified form)

**Comment by mah11** [comment]
updated links to other tickets

**Comment by mah11** [comment]
> "this gene has 2 transcript isoforms"
> is clearer

I agree, and ...

> but I'm not sure that this is correct

it is correct.

> isoform usually refers to the protein.

This is not accurate. Both transcripts and proteins can have isoforms.

> I don't know if 'transcript isoform' can only be used if the resulting protein is different.

No, any time more than one transcript is made from a given gene, they are transcript isoforms.

As for that Biostars link, it's old, and the participants don't sound all that clueful, but that said, "transcript variant" looks like it means the same thing as "transcript isoform". I haven't seed "transcript variant" used as much, and I'm reliably informed -- not just by whoever happens to respond to a question posted to an online forum -- that "transcript isoform" is current de-facto standard usage.

**Comment by kimrutherford** [comment]
I went with "transcript isoform ID".  I hope I got that right from the comments.

I've added it as an admin-only extension.  It should be available in Canto from tomorrow.

**Comment by kimrutherford** [comment]
> I've added it as an admin-only extension. It should be available in Canto from tomorrow.

Val, this needs testing at some point.  There's definitely no rush though.

**Comment by ValWood** [comment]
I needed to test this, now on my list!

**Comment by ValWood** [comment]
I  have been using with no problems. We still need to address isoform specific alles but that is in another ticket.
https://github.com/pombase/curation/issues/3616
